src/platforms/javascript/guides/react/features/component-tracking.mdx

```diff
@@ -1,12 +1,18 @@
 ---
-title: React Profiler
+title: Track React Components
 excerpt: ""
-description: "Learn how Sentry's React SDK exports a `withProfiler` higher-order component that attaches React related spans to the current active transaction on the scope."
+description: "Learn how Sentry's React SDK lets you monitor the rendering performance of your React components"
```

Comment:
Let's make these consistent if it makes sense. If not, then just make sure to add a period to this sentence.

```suggestion
description: "Learn how Sentry's React SDK allows you to monitor the rendering performance of your application and its components."
```